### ISSUE TITLE
Windows line ending in styleguidist.js causing build error in version 1.3.2

### ISSUE THREAD

**Issue by graemetait** [vue-styleguidist/vue-styleguidist#43]
This error seems to have crept back in.

```
$ npm run styleguide

> @ styleguide 
> vue-styleguidist server

env: node\r: No such file or directory
```

It might be worth configuring git to automatically fix this for you so that it doesn't happen again. https://help.github.com/articles/dealing-with-line-endings/

Thank you.

**Comment by rafaesc** [comment]
Hi @graemetait, thank you, 

I already did what you told me. The problem is already solved in the version 1.3.3, thanks for the patience.

**Comment by graemetait** [comment]
That's great, thanks for the speedy response!
